providers/flagd/src/test/java/dev/openfeature/contrib/providers/flagd/e2e/RunFlagdInProcessCucumberTest.java

```diff
@@ -12,8 +12,7 @@
 @IncludeEngines("cucumber")
 @SelectClasspathResource("features")
 @ConfigurationParameter(key = PLUGIN_PROPERTY_NAME, value = "pretty")
-@ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "dev.openfeature.contrib.providers.flagd.e2e.rpc")
 @ConfigurationParameter(key = GLUE_PROPERTY_NAME, value = "dev.openfeature.contrib.providers.flagd.e2e.process")
-public class RunCucumberTest {
+public class RunFlagdInProcessCucumberTest {
```

Comment:
Having 2 of these is how we run 2 suites @Kavindu-Dodan . (see the annotation above which filers on package name). 

The other is `RunFlagdRpcCucumberTest`.

Reply:
Nice 🙌 this is a good finding :)